server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/eml/noTextBodyButHtmlBody.eml

```diff
@@ -0,0 +1,90 @@
+Return-Path: <returnPath>
+Received: from localhost (localhost [127.0.0.1])
+	 by imap (Cyrus v2.2.13-Debian-2.2.13-19+squeeze3) with LMTPA;
+	 Tue, 14 Mar 2017 16:47:05 +0100
+X-Sieve: CMU Sieve 2.2
+Received: from localhost (localhost [127.0.0.1])
+	by test.linagora.com (Postfix) with ESMTP id 751A62685
+	for <test@localhost>; Tue, 14 Mar 2017 16:47:05 +0100 (CET)
+Received: from localhost (localhost [127.0.0.1])
+	by ci-openpaas.linagora.dc1 (Postfix) with ESMTP id 6DDAD4CB88E
```

Comment:
Anonimize and remove useless header... Your giving for free our internal mail infra for potential attackers...